packages/files-ui/cypress/tests/file-management-spec.ts

```diff
@@ -1,7 +1,7 @@
 import { binPage } from "../support/page-objects/binPage"
-import { homePage } from "../support/page-objects/homePage"
+import { click, homePage } from "../support/page-objects/homePage"
```

Comment:
Codacy mentions that click is unused
```suggestion
import { homePage } from "../support/page-objects/homePage"
```

Reply:
Ah forgot to push that, thanks